client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -121,8 +132,16 @@ export default React.createClass( {
 	},
 
 	getInstallButton() {
-		if ( this.hasInstallButton() ) {
-			return <PluginInstallButton { ...this.props } />;
+		if ( this.props.selectedSite && this.props.selectedSite.jetpack ) {
+			if ( this.hasOrgInstallButton() ) {
+				return <PluginInstallButton { ...this.props } />;
```

Comment:
It would be better to pass them explicitly, but I want to avoid refactoring that in this PR, since it wasn't introduced with it.